chatbot.py

```diff
@@ -5,16 +5,22 @@
 import databaseconnect
 from googleMapsApiModule import direction
 from googleMapsApiModule import geocoding
-
+import logging
+from MapBot.config import logger
```

Comment:
Why `Mapbot.config` instead of `config`? 
Also, while importing only use `import config`, use `config.logger` in the code when necessary

Reply:
Yes @vishakha-lall , I have created a new file `logger_config.py` and added have set the debug logging level there and also the `logger` function. :)